damus/Models/BookmarksManager.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  BookmarksManager.swift
+//  damus
+//
+//  Created by Joel Klabo on 2/18/23.
+//
+
+import Foundation
+
+class BookmarksManager {
+    static private let userDefaults = UserDefaults.standard
+    
+    static var bookmarks: [String] {
+        get {
+            return userDefaults.stringArray(forKey: "bookmarks") ?? []
```

Comment:
this needs to be keyed on pubkey

Reply:
I have a pretty dumb helper for that: pk_setting_key

Reply:
Oh, right, multiple accounts? I can update that 🤙